hypermodularization/what-is-hypermodular.md

```diff
@@ -0,0 +1,53 @@
+## What is "hypermodularization"?
+
+What I[^a] mean by this is a project (examples: [React](https://facebook.github.io/react/ 'Facebook Code: React'), [redux](http://redux.js.org/ 'redux')) that is explicitly designed with the expectation that a single dependency will **not** be self-contained. A developer-user (henceforth: user) will have to install several, perhaps even “many”[^b], dependencies in order to make full use of the project/framework or perhaps even to use it at all (as intended).
```

Comment:
I'd be interested in finding out what the numbers are there! Is there a way to do this?!